provider/aws/formation/rack.json

```diff
@@ -607,6 +607,26 @@
       "Default": "No",
       "AllowedValues": [ "Yes", "No" ]
     },
+    "DockerTlsCA": {
+      "Type": "String",
+      "Description": "Base64 encoded CA to expose docker in https mode",
+      "AllowedPattern" : ".+"
+    },
+    "DockerTlsCAKey": {
+      "Type": "String",
+      "Description": "Base64 encoded CA key to expose docker in https mode",
+      "AllowedPattern" : ".+"
+    },
+    "DockerTlsCert": {
+      "Type": "String",
+      "Description": "Base64 encoded TLS cert to expose docker in https mode",
+      "AllowedPattern" : ".+"
+    },
+    "DockerTlsKey": {
+      "Type": "String",
+      "Description": "Base64 encoded TLS key to expose docker in https mode",
+      "AllowedPattern" : ".+"
+    },
```

Comment:
the user will have to pass these values in the installation? what happens if it goes empty?

Reply:
User don't have to provide this, when user install a v2 rack or updates v2 rack to this version we will add it for them
https://github.com/convox/rack/blob/c2d83c5685d345c6b11fb7ae49d9f7ba2f19376d/provider/aws/system.go#L235
https://github.com/convox/rack/blob/c2d83c5685d345c6b11fb7ae49d9f7ba2f19376d/provider/aws/system.go#L541

if these values goes empty or invalid then v2 rack installation will fail

Reply:
@nightfury1204 can you remove the DockerTls* parameters from rack.json?
since we are generating when installing/upgrading I don't see the need to have them here.